app/controllers/dashboards_controller.rb

```diff
@@ -3,7 +3,6 @@ class DashboardsController < BaseController
   before_action :set_policy_with_context, except: [:refresh_dashboard]
   before_action :set_dashboard
   before_action :set_defence_requests, only: [:show, :closed]
-  before_action :set_dashboard_count, only: [:show, :closed]
```

Comment:
I agree with this idea. The show action should not be reserved for showing a particular type of dashboard.